sig/lrama/lexer/token/type.rbs

```diff
@@ -0,0 +1,15 @@
+module Lrama
+  class Lexer
+    class Token
+      attr_accessor type: Type
+      attr_accessor s_value: String
+      attr_accessor alias: String
+      attr_accessor keyword_init: bool
```

Comment:
`keyword_init` of `Struct.new` doesn't define `keyword_init` accessor on the class it creates, I noticed it after merge.

```ruby
>> s = Struce.new(:a, keyword_init: true)
>> s.new(a: 1).a
=> 1
>> s.new(a: 1).keyword_init
(irb):6:in `<main>': undefined method `keyword_init' for an instance of #<Class:0x0000000105b9df38>(keyword_init: true) (NoMethodError)
```